contracts/mocks/MockCTKN.sol

```diff
@@ -3,6 +3,9 @@ pragma solidity ^0.4.19;
 import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
 
 
+/**
+ *  A mock token used in testing.
+ */
 contract MockCTKN is MintableToken {
```

Comment:
If we have to change to individually capped tokens then also mocks would have to be changed from MintedToken to CappedToken

Reply:
Either way it would still need to be mintable.